include/eve/detail/shuffle_v2/simd/common/shuffle_l_fallback.hpp

```diff
@@ -20,4 +20,18 @@ requires(!abi_t<T, N>::is_wide_logical) && requires { shuffle_v2_core(x.bits(),
   // to mask and from mask + 2 each
   return kumi::tuple {to_logical(shuffled), eve::index<decltype(l)::value + 4>};
 }
+
+
+template<simd_value T, std::ptrdiff_t G, std::ptrdiff_t... I>
+EVE_FORCEINLINE auto
+shuffle_l_fallback_(EVE_SUPPORTS(cpu_), pattern_t<I...> p, fixed<G> g, T x, T y)
+{
+  // sse2 has no blend. Other should try blend/no blend options.
```

Comment:
do they ? Looks like they don't here or is doen elsewhere ?

Reply:
It's just not implemented :)